.travis.yml

```diff
@@ -1,7 +1,42 @@
 ---
-rvm:
-- 2.3.1
+sudo: false
 language: ruby
-bundler_args: "--without development integration"
-gemfile:
-- Gemfile
+bundler_args: "--without development"
+dist: trusty
+cache: bundler
+
+rvm: 2.3.3
+
+before_install:
+  - gem update --system # see https://github.com/bundler/bundler/issues/5357
+
+env:
+ - INSTANCE=ubuntu-12-04
+ - INSTANCE=ubuntu-14-04
+ - INSTANCE=ubuntu-16-04
+ - INSTANCE=centos-6
+ - INSTANCE=centos-7
+ - INSTANCE=debian-7
+ - INSTANCE=debian-8
+ - INSTANCE=fedora-24
+ - INSTANCE=fedora-25
+
+script:
+  -  bundle exec rake prepare_do_env kitchen
+
+matrix:
```

Comment:
maybe we should use a syntax like:
```
matrix:
  include:
  - rvm: 2.3.3
    bundler_args: "--without guard tools"
    script: bundle exec rake $SUITE
    env: SUITE=test:integration OS=default-ubuntu-1204 DOCKER=true
```
reference: https://github.com/chef/inspec/blob/master/.travis.yml#L15-L31